lib/zold/remotes.rb

```diff
@@ -224,13 +227,24 @@ def error(host, port = PORT)
       if_present(host, port) { |r| r[:errors] += 1 }
     end
 
+    def unerror(host, port = PORT)
+      raise 'Host can\'t be nil' if host.nil?
+      raise 'Port can\'t be nil' if port.nil?
+      raise 'Port has to be of type Integer' unless port.is_a?(Integer)
+
+      if_present(host, port) do |remote|
+        remote[:errors] -= 1 if remote[:errors] > 0
```

Comment:
In this case, instead of checking "> 0" it is better to use "positive?", same effect but clearer

Reply:
honestly, I think `> 0` is clearer than `positive?` it is not a huge difference but for sure one will have to think why greater than positive instead of `0`

Reply:
@lastk `> 0` it is clearer than `positive?` maybe because you know other languages (maybe C/C++) and you are used to it. But for a "True Rubyst"™ it is much better using `positive?`, and actually you can see that Trevis agrees with me

Reply:
@emilianodellacasa I only saw this comment now :( but I made the fix, my emacs was screaming at me about it too :P